src/OpenLoco/TownManager.cpp

```diff
@@ -28,7 +32,7 @@ namespace OpenLoco::TownManager
         dword_1135C38 = town;
         if (town != nullptr)
         {
-            town->var_34 += unk1;
+            town->populationCapacity += unk1;
```

Comment:
i think we can name unk1 as populationCapacity as well and unk3 can be named `numBuildings`. Which will also name var_38 as `numBuildings`

Reply:
I agree about unk1, I'm curious how did you figure out unk3?

Reply:
My thoughts are that it is increasing by 1 for each building and this function is called on town creation. There aren't going to be any other functions searching for all buildings of a town. createBuilding game command also passes `1` to this function and removeBuilding passes `-1`

Reply:
Done.